### ISSUE TITLE
[Accessibility] A11y_Manage.IIS.Net_Web Server Files Editor_Keyboard: By using the Tab/Shift+Tab, instead of going to next elememt keyboard focus is taking extra space(shifting the element to right)

### ISSUE THREAD

**Issue by Amlan454** [Microsoft/IIS.WebManager#204]
Environment Details:
URL: https://manage.iis.net/settings/servers
Browser: Edge(42.17134.1.0), Chrome(68.0.3440.106)
MAS ID Violated: MAS 2.1.2, MAS 2.4.3

Repro Steps:
1.Open "Microsoft IIS" application using the URL: https://manage.iis.net/
2.Download and install "Microsoft IIS Administration" application.
3.Tab till Microsoft IIS on top navigation bar and press enter
5.tab till web server and press enter
6.Tab till Files and press enter
7.Select any files from the list and press enter till to open html code page.(Ex-inetpub/custerr/en-US/401-1.htm/)
8.Navigate through the code by using the Tab/Shift+Tab key
9.Verify that whether user is able to come out from the code by using this key or not

Actual Result:
By using the Tab/Shift+Tab  key user is not able to come out from the coding part

Expected Result:
By using the Tab or Shift+Tab keyboard focus should come out from the code part 

User Impact:
Keyboard only users will face difficulty if the focus will stay in a particular region

Mas Reference:
https://microsoft.sharepoint.com/:w:/r/teams/msenable/_layouts/15/WopiFrame.aspx?sourcedoc={ebdc3fe1-09a9-4aa2-a29b-e97764774e6c}

Reference video:
[Bug_204_keyboard trap issue.pptx](https://github.com/Microsoft/IIS.WebManager/files/2338960/Bug_204_keyboard.trap.issue.pptx)

**Comment by jhkimnew** [comment]
We use Monaco editor and users can toggle the tab mode with Crtl-M hotkey. To change the input focus, users should hit first Ctrl-M and then hit Tab key.

See this guide for the detail information:
https://github.com/Microsoft/monaco-editor/wiki/Monaco-Editor-Accessibility-Guide

**Comment by Tanushagottipati** [comment]
GitHubTags: #A11yTCS; #A11yMAS; #DesktopWeb; #Win10; #A11y_Manage.IIS.Net; #Keyboard; #MAS2.1.2; #C+E-H; #.NETWebsites; #Chrome; #Edge; #MAS2.4.3

**Comment by prabhudevu** [comment]
**GitHubTags:**#A11ySev3;
